### PR TITLE
ignore whitespace. press enter with no input or input ended by whitespace 

### PR DESCRIPTION
**Pull request by tzzed**
this a fix of #106

```
genji> <press return, with or without whitespace>
genji> 
```

and 

```
genji> create table foo; <press return with whitespace after the ; >
... <press ';', then return>
genji>
```

both are fixed.

### CODE REVIEW

File: cmd/genji/shell/shell.go [hunk 1]
```diff
@@ -192,12 +202,13 @@ func (sh *Shell) executeInput(in string) error {
 		return sh.runCommand(in)
 	// If it ends with a ";" we can run a query
 	case strings.HasSuffix(in, ";"):
```
Comment (asdine) [inline cmd/genji/shell/shell.go:204]: I think we just need to ignore the query if it matches the condition we discussed in the issue.
Basically, I think all we need to do is this:
- trim the `in` variable to remove all withespaces
- ignore the command if `len(in) == 0`

File: cmd/genji/shell/shell.go [hunk 2]
```diff
@@ -192,12 +202,13 @@ func (sh *Shell) executeInput(in string) error {
...
-		sh.multiLine = false
-		sh.livePrefix = in
-		err := sh.runQuery(sh.query)
-		sh.query = ""
-		return err
+		return sh.QueryError(in)
+	//If we press enter without any query
+	case len(in) == 0:
+		return sh.QueryError(in)
+	// If it ends by espace "; "
```
Comment (asdine) [inline cmd/genji/shell/shell.go:209]: This check is not necessary if we trim `in`

File: cmd/genji/shell/shell.go [hunk 3]
```diff
@@ -184,6 +184,16 @@ func (sh *Shell) execute(in string) {
 	}
 }
 
+// Query Error return an error if we press return with no input, or ended by whitespace
+func (sh *Shell) QueryError(in string) error {
```
Comment (asdine) [inline cmd/genji/shell/shell.go:188]: The name of this function doesn't seem to have anything to do with the content of the function, does it?

File: cmd/genji/shell/shell.go (hunk 1 above)
Comment (tzzed) [reply cmd/genji/shell/shell.go:195]: Indeed.

**Comment by tzzed** [comment]
implementation of @asdine's suggestion for a simpler version.

**Comment by asdine** [comment]
@tzzed I can't merge your PR, you need to [rebase](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request) it

**Pull request closed without merging**